backend/src/main/scala/cromwell/backend/BackendActor.scala

```diff
@@ -0,0 +1,85 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging, ActorRef}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendActor._
+import cromwell.backend.model.{JobDescriptor, WorkflowDescriptor}
+
+object BackendActor {
+
+  // Commands
+  sealed trait BackendActorMessage
+  case object BeforeAll extends BackendActorMessage
+  case class CreateJobExecutorActor(jobDescriptor: JobDescriptor) extends BackendActorMessage
+  case object AfterAll extends BackendActorMessage
+  case object Validate extends BackendActorMessage
```

Comment:
It's hidden now but you mentioned caching before as something that needs to be added. Did you mean for the caching writing or reading

Did you foresee other commands being necessary for that/those?